spec/lib/tasks/dev_rake_spec.rb

```diff
@@ -25,14 +25,19 @@
     prev_verified = nil
     prev_scrypt_cost = nil
     prev_pending_in_usps = nil
+    usps_request_delay_ms = nil
+    max_num_attempts = nil
 
     before(:each) do
+      allow(IdentityConfig.store).to receive(:usps_mock_fallback).and_return(false)
+      max_num_attempts = ENV['MAX_NUM_ATTEMPTS']
```

Comment:
I think we could simplify all the before/after blocks if the before block stubbed `ENV`:


```
let(:env) { { 'NUM_USERS' => num_users } } # etc

before do
  stub_const('ENV', env)
end
```

and then we don't need the `after` block at all to restore

Reply:
Nice. Updated in 4c5ee1edb